overlord/fdestate/dbx_test.go

```diff
@@ -817,13 +849,42 @@ func (s *fdeMgrSuite) TestEFIDBXUpdatePostUpdateResealFailed(c *C) {
 	c.Check(resealForDBUPdateCalls, Equals, 1)
 	c.Check(resealForBootChainsCalls, Equals, 0)
 
+	chgs := st.Changes()
+	c.Assert(chgs, HasLen, 1)
+	chg := chgs[0]
+	c.Check(chg.IsReady(), Equals, false)
+	tsks := chg.Tasks()
+	c.Assert(tsks, HasLen, 2)
+	c.Assert(tsks[0].Kind(), Equals, "efi-secureboot-db-update-prepare")
+	c.Assert(tsks[1].Kind(), Equals, "efi-secureboot-db-update")
+
 	st.Unlock()
 	defer st.Lock()
 
-	// cleanup triggers post update reseal
+	// first reach a known steady state
+	iterateUnlockedStateWaitingFor(st, func() bool {
```

Comment:
nitpick: given we need to do this anyway, wouldn't it be simpler to not call Loop and instead have the helper call the manager Ensure directly? as I mentioned yesterday we do have test using that style too sometimes

Reply:
per discussion from mattermost, overlord `Loop()` is already running since the DBX API calls need it for tasks to reach a certain stage before returning (specifically the efi-secureboot-db-update-prepare task). I suppose I could start a goroutine and call Settle() in there, but it's equally hacky and inconvenient.